src/profile.vala

```diff
@@ -123,14 +123,11 @@ class Profile: Object
   protected Action create_enabled_action (Bluetooth bluetooth)
   {
     var action = new SimpleAction.stateful ("bluetooth-enabled",
-                                            null,
+                                            VariantType.BOOLEAN,
                                             new Variant.boolean (bluetooth.enabled));
 
-    action.activate.connect (()
-        => action.change_state (new Variant.boolean (!action.get_state().get_boolean())));
-
-    action.change_state.connect ((action, requestedValue)
-        => bluetooth.try_set_enabled (requestedValue.get_boolean()));
+    action.activate.connect ((action, state)
```

Comment:
Explain this?

Why did action.activate.connect get replaced with action.change_state.connect functions and action.change_state.connect gets completely removed

Reply:
> Explain this?
> 
> Why did action.activate.connect get replaced with action.change_state.connect functions and action.change_state.connect gets completely removed

Please see https://github.com/AyatanaIndicators/ayatana-indicator-bluetooth/pull/1#discussion_r496245536